src/applications/pensions/tests/pensions.cypress.spec.js

```diff
@@ -14,7 +14,7 @@ const fillAdditionalSources = ({ additionalSources }) => {
   cy.findByText(/add another/i).click();
   cy.findByLabelText(/source/i).type(name);
   cy.findByLabelText(/amount/i).type(amount);
-  cy.findByText(/save/i).click();
+  cy.findByText(/^Save$/).click();
```

Comment:
The heading now contains the text `Your application will be saved on every change.`, so this selector matched both the heading _and_ the button it should match. Making this selector more specific finds only the button again.